### ISSUE TITLE
molecular transformer side texture limits mip level from 4 to 3

### ISSUE THREAD

**Issue by Meduris** [Trinsdar/Advanced-Solars-Classic-Edition#7]
just noticed this message in my startup log with advanced solars included:

> [17:45:05] [Worker-Main-7/WARN] [minecraft/TextureAtlas]: Texture advanced_solars:block/molecular_transformer/side with size 64x8 limits mip level from 4 to 3

other textures might be affected too, as from my experience, this warning is only shown the first time the mip level is limited to a lower level

Advanced Solars Classic Version: 2.0.1 for Minecraft 1.19.2

**Comment by Trinsdar** [comment]
not sure what exactly this means, but the texture it's referring to is an animated texture with 8 frames, and is 64x64

**Comment by Meduris** [comment]
Yeah, I just noticed that. Not sure what exactly minecraft is complaining about in this case then.

I'll try to look into it this evening and test a few things and will report back in case I find out whats causing this

**Comment by Meduris** [comment]
Ok, so in a short test I noticed that the warning no longer appears after removing the ``"height": 8`` part from [side.png.mcmeta](https://github.com/Trins-mods/Advanced-Solars-Classic-Edition/blob/master/src/main/resources/assets/advanced_solars/textures/block/molecular_transformer/side.png.mcmeta)

Apparently minecraft by default already expects each animation frame to use a square and which in your case automatically results in 64x64 px textures for each frame. Might be that something about the height parameter that is not quite correct, but from what I read it is only inteded for non square frame textures

**Comment by Trinsdar** [comment]
Ah ok
